### ISSUE TITLE
Historical Position Data

### ISSUE THREAD

**Issue by flipperbw** [BurntSushi/nflgame#304]
Would it be possible to add this to the database? It's tough querying for older or changed players when they move positions, or get listed as UNK after playing years at a certain position. It would also help for when a WR throws a pass during a game (if you only want true QB stats).

Thanks.

**Comment by BurntSushi** [comment]
> Would it be possible to add this to the database?

Yes, it is possible. But will it happen? No. Because it requires a lot of work.

**Comment by flipperbw** [comment]
No problem, I figured as much. I spent a lot of yesterday doing just that. I parsed the data from http://www.footballoutsiders.com/stats/snapcounts. They only list the first initial and uniform number, but that is enough to get about 80% of player_id's matched up. I'm going to finish matching up today hopefully. The data has position information as well as snaps, which is really useful. Here's what the DB looks like in case anyone is wondering:

` player_id  | first_initial |   last_name    | player_num | team | position | started | total_snaps | off_snaps | off_snap_pct | def_snaps | def_snap_pct | st_snaps | st_snap_pct | week | year
------------+---------------+----------------+------------+------+----------+---------+-------------+-----------+--------------+-----------+--------------+----------+-------------+------+------
 00-0021678 | T             | Romo           |          9 | DAL  | QB       | t       |          71 |        71 |          100 |         0 |            0 |        0 |           0 |    1 | 2015
 00-0026498 | M             | Stafford       |          9 | DET  | QB       | t       |          47 |        47 |          100 |         0 |            0 |        0 |           0 |    1 | 2015
 00-0027981 | K             | Rudolph        |         82 | MIN  | TE       | t       |          54 |        52 |           93 |         0 |            0 |        2 |           9 |    1 | 2015
`

**Comment by paulz1** [comment]
Sorry to disturb...
I'm very interesting in a such base.
Does your base is accessible anywhere... or may be better is it possible to get the code to generate this base by myself.

P.S.: BurntSushi, sorry to write here. But it seems that there is no Private Messages in github anymore.
